File: 2.server/src/main/java/ru/vitaliy/petrov/server/forms/requests/userprofile/VehicleInsurancePolicyCreationRequest.java
package ru.vitaliy.petrov.server.forms.requests.userprofile;

import lombok.Data;

import javax.validation.constraints.NotEmpty;
import javax.validation.constraints.NotNull;
import java.sql.Date;

@Data
public class VehicleInsurancePolicyCreationRequest {

    @NotNull
    @NotEmpty
    private final String vehicleInsuranceCompany;

    @NotNull
    @NotEmpty
    private final String vehicleInsurancePolicyNumber;

    @NotNull
    private final Date vehicleInsurancePolicyExpirationDate;
}

File: 3.client/app/src/main/java/com/petrov/vitaliy/caraccidentapp/domain/models/requests/user/VehicleInsurancePolicyCreationRequest.kt
package com.petrov.vitaliy.caraccidentapp.domain.models.requests.user

import java.sql.*

data class VehicleInsurancePolicyCreationRequest(
    val vehicleInsuranceCompany: String,
    val vehicleInsurancePolicyNumber: String,
    val vehicleInsurancePolicyExpirationDate: Date
)
